app/Console/Commands/Cloud/UsagePingCommand.php

```diff
@@ -0,0 +1,99 @@
+<?php
```

Comment:
Please include a test for this command.

Reply:
Preferably a test that doesn't break travis every other commit*

Reply:
woah, now you're asking for too much...